Client/Frontend/Browser/TabManager.swift

```diff
@@ -740,6 +740,10 @@ extension TabManager {
             let tab = addTab()
             selectTab(tab)
         }
+        // Always make sure there is a single normal tab.
+        if normalTabs.isEmpty {
+            addTab()
+        }
```

Comment:
No, because if we previously had a private tab selected, we don't want to lose that selection.